### PR TITLE
English: Koppelingen

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Translates the koppelingen page to integrations

### CODE REVIEW

File: source/localizable/koppelingen.html.erb [hunk 1]
```diff
@@ -2,10 +2,11 @@
 title:
   nl: Koppelingen met HR Systemen
   de: Schnittstellen mit Quellsystemen
-  en: Integrations
+  en: Integrations with met HR systems
```
Comment (fatboypunk) [inline source/localizable/koppelingen.html.erb]: with met

File: source/localizable/koppelingen/_koppelingen.en.md.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Defacto is using a standard HR Importer. Via the HR importer, CAPP can import the data from the source systems such as Beaufort, People Soft, SAP and many other HR systems.
```
Comment (fatboypunk) [inline source/localizable/koppelingen/_koppelingen.en.md.erb]: `Via` -> `through`

Comment (fatboypunk) [inline source/localizable/koppelingen/_koppelingen.en.md.erb]: `standard HR Importer. Via the HR importer, ` would like some other frasing here

File: source/localizable/koppelingen/_koppelingen.en.md.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+We treat our HR importer as one of our products, which ensures that you will always have the right version of the importer that fits with the correct version of CAPP. What is more, Defacto has established an integration with numerous systems:
+
+### Verpleegkundigen &amp; Verzorgend Nederland (V&amp;VN), the Dutch professional Union for nurses
+
+With V&amp;VN the accredited E-learning courses that your employees take in CAPP are automatically registered within CPD-online. This ensures that you or your employees don’t have to carry out administrative tasks twice.
+
+### Learning Tools Interoperability (LTI)
+
+All E-learning content  that supports LTI integration and is hosted somewhere else, can   now be viewed in CAPP. As a result, you are no longer dependent on one supplier and can decide for yourself from where you get your E-learning content. In this way the content from an external supplier can be viewed in CAPP and the generated results can be displayed in the CAPP-environment.
```
Comment (fatboypunk) [inline source/localizable/koppelingen/_koppelingen.en.md.erb]: content  that 
double space

Comment (fatboypunk) [inline source/localizable/koppelingen/_koppelingen.en.md.erb]: can   now
double space

Comment (fatboypunk) [inline source/localizable/koppelingen/_koppelingen.en.md.erb]: maybe we should get all the you/yours out of this text?

File: source/localizable/koppelingen.html.erb (hunk 1 above)
Comment (RonaldDijkstra) [inline source/localizable/koppelingen.html.erb]: How did I miss that?

File: source/localizable/koppelingen/_koppelingen.en.md.erb (hunk 2 above)
Comment (RonaldDijkstra) [inline source/localizable/koppelingen/_koppelingen.en.md.erb]: It's now: 

> Through our HR importer, we can import data from other source systems, such as Beaufort, People Soft, SAP and many other HR systems.

File: source/localizable/koppelingen/_koppelingen.en.md.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+### Verpleegkundigen &amp; Verzorgend Nederland (V&amp;VN), the Dutch professional Union for nurses
+
+With V&amp;VN the accredited E-learning courses that your employees take in CAPP are automatically registered within CPD-online. This ensures that you or your employees don’t have to carry out administrative tasks twice.
+
+### Learning Tools Interoperability (LTI)
+
+All E-learning content that supports LTI integration and is hosted somewhere else can be viewed in CAPP. As a result, organisations are no longer dependent on just one supplier and can decide from where to get E-learning content. In this way the content from an external supplier can be viewed in CAPP and the generated results can be displayed in the CAPP-environment.
+
+### Medisch Onderwijs (MO), a platform for medical e-learning applications
```
Comment (sn3p) [inline source/localizable/koppelingen/_koppelingen.en.md.erb]: `E-learning` (for consistency)

**Pull request merged**